tests/5.0/target/test_target_has_dev_addr.c

```diff
@@ -0,0 +1,57 @@
+//===------ test_target_has_dev_addr.c--------------------------------------===//
+//
+// OpenMP API Version 5.0 Nov 2018
+//
+// Tests the target directive with parallel for + device + has_device_addr clauses
```

Comment:
`has_device_addr` was added to the 5.1 specification, please move it to the 5.1 directory.